lib/loaders/loaders_with_authentication/impulse.js

```diff
@@ -4,6 +4,44 @@ import { gravityLoaderWithAuthenticationFactory, impulseLoaderWithAuthentication
 
 const { IMPULSE_APPLICATION_ID } = process.env
 
+/**
+ * This optimizes conversations requests by removing requests that are only interested in the `total_count` and instead
+ * rely on another request to provide that data.
+ */
+function optimize(keys) {
+  if (keys.length === 1) {
+    /**
+     * If there’s only 1 request, then it doesn’t matter if it’s a counter or not, we can’t optimize it onto any
+     * other request anyways, so just resolve the original key.
+     *
+     * TODO It could well be that all optimizations will perform this same check, if so just move this into the
+     *      loader factory.
+     */
+    return { keys }
+  }
+  /**
+   * NOTE We only ever query convos of 1 user (the authenticated one) for the duration of 1 query execution, if we
```

Comment:
One other thing to maybe keep in mind that if we're ever filtering conversations by _more_ than just one user, `total_count` there might mean something else. #minor